collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh

```diff
@@ -605,6 +633,25 @@ while getopts ":d:s:r:fph" opt; do
   esac
 done
 
+# Check upload has required flags
+if [ -n "${UPLOAD}" ]; then
+  if [ -z "${LOCATION}" ]; then
+    techo "Option -u requires -l to be set."
+    exit 1
+  else
+    if [ "${LOCATION}" == "us" ] || [ "${LOCATION}" == "emea" ]; then
```

Comment:
We should probably default to "us" and make "emea" guys explicitly override for "GDPR" compliance? Other way round is fine too.

Reply:
I had the same thought.

Reply:
While there are "us" and "emea" names for the fqdn, these aren't real anymore. The naming was carried over from the Microfocus days when we had a physical FTP server located in the US. Currently all of the domains point to the same system which is hosted in Frankfurt, Germany.

Reply:
So should we just set it to "us" and then leave it? Then I could take the option out completely and hard-code the URL.

Reply:
Removing it or having emea as the only option would prevent confusion. Its possible we may have one in the US again to meet some customers requirements but I haven't heard any chatter about that in a while.